README.md

```diff
@@ -146,21 +145,23 @@ the config in the following format:
 ```php
 $config = [
     'di' => [
-        'logger' => [
-            'class'     => Logger::class,
-            'singleton' => true,
-            'arguments' => [
-                StdoutLogger::class
+        'services' => [
+            'logger' => [
+                'class'     => Logger::class,
+                'singleton' => true,
+                'arguments' => [
+                    StdoutLogger::class
```

Comment:
@tomphp 

Apologies for the nitpicking, but for consistency should we always add a trailing `,` in array initializations?